docs/concepts/time-and-windows-in-ksql-queries.rst

```diff
@@ -180,36 +187,26 @@ windows per record key.
 
 When using windows in your KSQL queries, aggregate functions are applied only
 to the records that occur within a specific time window. Records that arrive
-late are handled as you might expect: although the time window they belong to
-has expired, the late records are still associated with the correct window.
+out-of-order are handled as you might expect: although the time window they belong to
+has expired, the out-of-order records are still associated with the correct window.
 
-You can specify a retention period for the window in your KSQL queries. This
-retention period controls how long KSQL waits for out-of-order or late-arriving
-records for a given window. If a record arrives after the retention period of a
-window has passed, the record is discarded and isn't processed in that window.
+You can specify a *grace period* for the window in your KSQL queries. This

```

Comment:
How do you specify the 'grace period'?  Is there an example in `grace-period-late-events.rst`?  Could you add it into the review to help my understanding?

Reply:
@mjsax , @big-andy-coates 's questions are for the KSQL docs that incorporate content from Streams docs related to "grace period". @JimGalasyn and I used `includes` to share content directly on the tip ("grace period vs. retention time") and the paragraph above it starting with "Out-of-order records are always possible in the real world ..", but the rest of the content we had to each write separately to be specific to KSQL or Streams.

@big-andy-coates 's questions are on the KSQL specific content. Can you review and clarify for Andy,  perhaps provide an example, and/or let us know how we should change what we have here?

Reply:
Not sure if KSQL exposed "grace period" to the user already? It was added to Kafka Streams in 2.1 release, but maybe there is no equivalent in KSQL yet -- KSQL also does not yet support `suppress()` operator.

Reply:
@JimGalasyn, see ^^ @mjsax comment, maybe explore whether this is exposed yet? If not, we can re-introduce here when it is ..?

Reply:
@mjsax @big-andy-coates @londoncalling  Good feedback, I'll post another PR in a minute.